app/views/users/show.html.erb

```diff
@@ -24,15 +24,103 @@
     }
     <% end %>
   </style>
-  <div class="user-profile-header" style="<%= user_colors_style(@user) %>" itemscope itemtype="http://schema.org/Person" itemprop="mainEntity">
-    <meta itemprop="url" content="https://dev.to/<%= @user.username %>">
+
+<script type="application/ld+json">
+{
+    "@context": "http://schema.org",
+    "@type": "Person",
+    "mainEntityOfPage": {
+      "@type": "WebPage",
+      "@id": "https://dev.to/<%= @user.username %>"
+    },
+    "url": "<%= user_url(@user) %>",
+    "sameAs": [
+      <% if @user.twitter_username? %>
+        "https://twitter.com/<%= @user.twitter_username %>",
+      <% end %>
+      <% if @user.github_username? %>
+        "https://github.com/<%= @user.github_username %>",
+      <% end %>
+      <% if @user.mastodon_url? %>
+        "<%= @user.mastodon_url %>",
+      <% end %>
+      <% if @user.facebook_url? %>
+        "<%= @user.facebook_url %>",
+      <% end %>
+      <% if @user.linkedin_url? %>
+        "<%= @user.linkedin_url %>",
+      <% end %>
+      <% if @user.behance_url? %>
+        "<%= @user.behance_url %>",
+      <% end %>
+      <% if @user.stackoverflow_url? %>
+        "<%= @user.stackoverflow_url %>",
+      <% end %>
+      <% if @user.dribbble_url? %>
+        "<%= @user.dribbble_url %>",
+      <% end %>
+      <% if @user.medium_url? %>
+        "<%= @user.medium_url %>",
+      <% end %>
+      <% if @user.gitlab_url? %>
+        "<%= @user.gitlab_url %>",
+      <% end %>
+      <% if @user.instagram_url? %>
+        "<%= @user.instagram_url %>",
+      <% end %>
+      <% if @user.twitch_username? %>
+        "<%= @user.twitch_url %>",
+      <% end %>
+      <% if @user.website_url? %>
+        "<%= @user.website_url %>"
+      <% end %>
+    ],
+    "image": "<%= ProfileImage.new(@user).get(width: 320) %>",
+    "name": "<%= @user.name %>",
+    <% if @user.email_public == true %>
```

Comment:
You can just do `@user.email_public` here because it returns a Boolean value. The `== true` is redundant

Reply:
I had added this as a test and never removed this. 🤦‍♀️ Good catch!